src/pcapi/routes/native/serialization/authentication.py

```diff
@@ -0,0 +1,10 @@
+from pydantic import BaseModel
+
+
+class SigninBodyModel(BaseModel):
```

Comment:
Un body peut être utilisé en requête ou réponse, du coup c'est pas clair. Il vaudrait mieux renommer :
```suggestion
class SigninRequestModel(BaseModel):
```